frontend/src/fsrs_cards.tsx

```diff
@@ -121,16 +126,17 @@ const FSRSCards: React.FC<FSRSCardsProps> = ({
         alphagram: currentCard.alphagram?.alphagram ?? "",
         nextScheduled: scoreResponse.nextScheduled!,
         cardRepr: JSON.parse(
-          new TextDecoder().decode(scoreResponse.cardJsonRepr)
+          new TextDecoder().decode(scoreResponse.cardJsonRepr),
```

Comment:
Trying out a new text editor and it seems like this auto-formatted, not totally sure why